docs/usage/editing-paths.md

```diff
@@ -0,0 +1,78 @@
+
+### Waypoint Scope
+
+Applies this constraint between at that specific waypoint.
+
+### Segment Scope
+
+Applies this constraint between waypoints (intuitive example: think constraining your trajectory as a straight line between two waypoints)
+
+### Waypoint + Segment Scope
+
+Like a segment scope, but including segments (intuitive example: think of zero angular velocity)
+
+| Name                        | Description                                                                                                                                                                                                                                          | Waypoint Scope | Segment Scope |
+| --------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- | ------------- |
+| Waypoint Velocity Direction | Constrains the robot's direction of travel at a waypoint.                                                                                                                                                                                            | Yes            | No            |
+| Waypoint Zero Velocity      | Stop at a certain point                                                                                                                                                                                                                              | Yes            | No            |
+| Max Velocity                | Constrains the max velocity to a specific value throughout multiple waypoints/segments This generally refers to the Maximum Velocity of robot chassis in m/s. A default value can be defined in Robot Config ([Getting Started](../robot-config.md)) | Yes            | Yes           |
+| Zero Angular Velocity       | Ensures zero angular velocity through the waypoint and/or segment scope.                                                                                                                                                                             | Yes            | Yes           |
+| Straight Line               | Tells the solver to draw a straight line as the path between two immediate waypoints.                                                                                                                                                                | No             | Yes           |
+
+## Obstacles
+
+Obstacles allow the robot to avoid a certain area of the field throughout the path. For example, in the 2023 Charged Up game, you might want to avoid crashing into the charge station. Obstacles makes it to define where you want the robot to start and end without unnecessary intermediary waypoints.
```

Comment:
```suggestion
Obstacles allow the robot to avoid a certain area of the field throughout the path. For example, in the 2023 Charged Up game, you might want to avoid crashing into the charge station. Obstacles makes it easy to define where you want the robot to start and end without unnecessary (and performance-impacting) intermediary waypoints.
```